### PR TITLE
Strict Parsing Decimal Documentation

### PR DESCRIPTION
**Pull request by AdrianOlosutean**

### CODE REVIEW

**Review by zakera786 — APPROVED** [review]

File: _docs/2.0.0/usage/schema.md [hunk 1]
```diff
@@ -271,6 +271,7 @@ Here are the recognized ones with the description of their purpose (with detaile
 | [grouping_separator](#grouping_separator) | any numeric type            | Character to mark boundaries between orders of magnitude, usually to mark thousands, millions etc.                                                    | *\_*            | *,*                                          |
 | [minus_sign](#minus_sign)                 | any numeric type            | Character to mark the number is negative.                                                                                                             | *N*             | *-*                                          |
 | [allow_infinity](#allow_infinity)         | float & double              | Flag indicating if the column accepts infinity as a value (and positive/negative numbers which are too large are converted to *infinity*/*-infinity*) | *true*          | *false*                                      |
+| [strict_parsing](#strict_parsing)         | decimal                     | Flat indicating strict parsing on the provided decimal value. Strict parsing rejects a value(including default) with a larger scale than allowed      | *true*          | *false*                                      |
```
Comment (benedeki) [inline _docs/2.0.0/usage/schema.md:274]:
```suggestion
| [strict_parsing](#strict_parsing)         | decimal                     | Flag indicating if strict parsing should be done on the input value. Strict parsing rejects a value with a more decimal value than the defined scale of the decimal       | *true*          | *false*                                      |
```

File: _docs/2.0.0/usage/schema.md [hunk 2]
```diff
@@ -370,6 +371,15 @@ valid value, instead of failing with casting error ([see here]({{ site.baseurl }
...
 that are too large are converted to *infinity* and *-infinity*, respectively.
 
+### strict_parsing
+
+<sup>Supported by types:** [Decimal](#decimal)</sup>
+
+Flat indicating strict parsing on the provided decimal value.
+Strict parsing rejects a value(including default) with a larger scale than allowed with casting error
+([see here]({{ site.baseurl }}/docs/{{ page.version }}/usage-errcol)). For example for Decimal(X,2), the values with
+longer scale(like 10.12345, 0.1234) will rejected
```
Comment (benedeki) [inline _docs/2.0.0/usage/schema.md:381]:
```suggestion
Strict parsing rejects a value with more decimal places than the scale (second number) of the field's Decimal type definition. 
This results in a casting error in error column ([see here]({{ site.baseurl }}/docs/{{ page.version }}/usage-errcol)). For example for Decimal(X,2), the values with
longer scale (like 10.12345, 0.1234) will be rejected .
Default value have to fit the scale as well in the case of strict parsing.
```

File: _docs/2.0.0/usage/schema.md [hunk 3]
```diff
@@ -597,6 +607,7 @@ The value used when _explicit default_ was not defined in the schema:
 cannot be a `default` for the type [`Short`](#short), or _"∞"_ if `allow_infinity` is _"false"_ for [`Double`](#double)/
 [`Float`](#float))
 - If it's a type supporting [`pattern`](#pattern) and it is defined, the default value has to adhere to the `pattern`
+- If 'strict_parsing' is enabled for [`Decimal`](#decimal), the decimal scale should fit into the specified type scale
```
Comment (benedeki) [inline _docs/2.0.0/usage/schema.md:610]:
```suggestion
- If 'strict_parsing' is enabled for [`Decimal`](#decimal), the number of decimal places has to fit into the type's scale
```

**Review by benedeki — CHANGES_REQUESTED** [review]

File: _docs/2.0.0/usage/schema.md [hunk 4]
```diff
@@ -370,6 +371,15 @@ valid value, instead of failing with casting error ([see here]({{ site.baseurl }
 The string representing infinity on input is *"∞"* and *"-∞"* respectively. Positive and negative numbers with values 
 that are too large are converted to *infinity* and *-infinity*, respectively.
 
+### strict_parsing
+
+<sup>Supported by types:** [Decimal](#decimal)</sup>
+
+Flat indicating strict parsing on the provided decimal value.
```
[the remark this answers is not in the document]
Comment (AdrianOlosutean) [reply _docs/2.0.0/usage/schema.md:378]: `Exercised` sounds weird for me. I would say `applied`

[the remark this answers is not in the document]
Comment (benedeki) [reply _docs/2.0.0/usage/schema.md]: Applied it good 👍

File: _docs/2.0.0/usage/schema.md [hunk 5]
```diff
@@ -271,6 +271,7 @@ Here are the recognized ones with the description of their purpose (with detaile
 | [grouping_separator](#grouping_separator) | any numeric type            | Character to mark boundaries between orders of magnitude, usually to mark thousands, millions etc.                                                    | *\_*            | *,*                                          |
 | [minus_sign](#minus_sign)                 | any numeric type            | Character to mark the number is negative.                                                                                                             | *N*             | *-*                                          |
 | [allow_infinity](#allow_infinity)         | float & double              | Flag indicating if the column accepts infinity as a value (and positive/negative numbers which are too large are converted to *infinity*/*-infinity*) | *true*          | *false*                                      |
+| [strict_parsing](#strict_parsing)         | decimal                     | Flag indicating if strict parsing should be done on the input value. Strict parsing rejects a value with a higher decimal value the defined scale     | *true*          | *false*                                      |
```
Comment (benedeki) [inline _docs/2.0.0/usage/schema.md:274]:
```suggestion
| [strict_parsing](#strict_parsing)         | decimal                     | Flag indicating if strict parsing should be done on the input value. Strict parsing rejects a value with more decimal places than the defined scale     | *true*          | *false*                                      |
```
Sorry, the previous suggestion didn't make sense there.

**Review by benedeki — APPROVED** [review]

**Pull request merged**